### DISCUSSION TITLE
version of visNetwork in "suggests" needs to be bumped (v2.1.2 works)

### DISCUSSION THREAD

**Discussion by corypdavis** [ropensci/targets#1152] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hi there, thankyou for this awesome package.  I just found a minor issue with an easy fix:

version 2.0.9 of visNetwork doesn't support the zoomSpeed argument used in class_visnetwork.R.  I have successfully tested with visNetwork 2.1.2

**Comment by wlandau** [comment]
Thanks for mentioning this, should be fixed now.
